### PR TITLE
Use v5 types on mint queries

### PR DESCRIPTION
**Pull request by fernandofcampos**
## Purpose of Changes and their Description

Params type was updated at mint/v5, but Queries and Tx were still using the old definition from v1beta1. 

This PR updates query.proto and tx.proto

## Are these changes tested and documented?

- [ ] If tested, please describe how. If not, why tests are not needed.
- [ ] If documented, please describe where. If not, describe why docs are not needed.
- [ ] Added to `Unreleased` section of `CHANGELOG.md`?

## Still Left Todo

*Fill this out if this is a Draft PR so others can help.*

### CODE REVIEW

File: x/mint/api/mint/v1beta1/codec.go
Comment (relyt29) [inline x/mint/api/mint/v1beta1/codec.go:1]: why are these being deleted shouldn't they still exist per @amimart's PRs to keep them around to be able to query old versions

File: x/mint/module/module.go [hunk 1]
```diff
@@ -52,7 +51,6 @@ func (AppModuleBasic) RegisterLegacyAminoCodec(cdc *codec.LegacyAmino) {
 // RegisterInterfaces registers the module's interface types
 func (b AppModuleBasic) RegisterInterfaces(r cdctypes.InterfaceRegistry) {
 	types.RegisterInterfaces(r)
-	v1beta1.RegisterInterfaces(r)
```
Comment (relyt29) [inline x/mint/module/module.go]: should this be deleted? I thought we were keeping old msg servers and query servers around

File: x/mint/api/mint/v1beta1/codec.go
Comment (fernandofcampos) [reply x/mint/api/mint/v1beta1/codec.go:1]: proto gen deleted that file. I didn't like it either.

I can restore manually and ser if it works.

File: x/mint/module/module.go (hunk 1 above)
Comment (fernandofcampos) [reply x/mint/module/module.go]: It had to be removed because the definition was deleted by proto gen.

File: x/mint/api/mint/v1beta1/codec.go
Comment (relyt29) [reply x/mint/api/mint/v1beta1/codec.go:1]: yah thats what I've been doing on previous PRs just putting it back manually

**Review by kpeluso — DISMISSED** [review]

File: x/mint/api/mint/v1beta1/codec.go
Comment (fernandofcampos) [reply x/mint/api/mint/v1beta1/codec.go:1]: Done

**Review by kpeluso — APPROVED** [review]

**Pull request merged**